### ISSUE TITLE
Toast group is getting cut off at 320 by 256 px

### ISSUE THREAD

**Issue by ranaJPMChase** [jpmorganchase/salt-ds#4704]
### Latest version

- [x] I have tested the latest version

### Description

Toast group and toast are getting cut off in 320 by 256 px screen for WCAG reflow checkpoint

### Steps to reproduce

have status error toast in toast group on screen right bottom placement
make screen 320 by 256 px 
Now check if toast is coming fully on screen. Is it getting cut or fully coming in middle of screen ?

### Expected behavior

Toast should not get cut and should appear properly in middle of screen.

### Package name(s)

Core (@salt-ds/core), Lab (@salt-ds/lab)

### Package version(s)

 "@salt-ds/core": "^1.40.0","@salt-ds/lab": "1.0.0-alpha.60",

### Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [ ] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.

**Comment by origami-z** [comment]
Can't reproduce this. Toast group would stay on bottom right, but not cut off on smaller viewport. Please provide minimal reproducible example. You're likely not setting appropriate width yourself, which doesn't come from Salt.

**Comment by ranaJPMChase** [comment]
@origami-z Please make sure to make screen 320 by 256 px its not about mobile screen but more about ADA compliance. You have to make sure that content is not getting cut off at 320px.
I was able to reproduce above issue with this code

`import { ToastGroup } from "@salt-ds/lab";
import React from "react";
import { Button, Toast, ToastContent, Text } from "@salt-ds/core";
import { CloseIcon } from "@pepper/icons";

const POC = () => {
  return <ToastGroup>
    <Toast status="error">
      <ToastContent>
        <div>
          <Text>
            <strong>System error</strong>
          </Text>
          <div>Connection timed out. Failed to retrieve data. </div>
        </div>
      </ToastContent>
      <Button appearance="transparent" aria-label="Dismiss">
        <CloseIcon aria-hidden />
      </Button>
    </Toast>
  </ToastGroup>
}

export default POC;`

Note: Please make sure that screen is 320 by 256 pixels. Thanks

**Comment by origami-z** [comment]
Toast group is a lab component, which is not currently a priority

**Comment by jake-costa** [comment]
Confirmed this is no longer an issue. Width needed to be set.
